gradle/dependencies.gradle

```diff
@@ -1,17 +1,18 @@
 ext.versions = [
-    kotlin: '1.3.61',
-    kotlinCoroutines: '1.3.3',
+    kotlin: '1.3.71',
+    kotlinCoroutines: '1.3.5',
     compileSdk: 29,
     idea: '2019.2',
+    ideaKotlin: '1.3.61',
     dokka: '0.9.18',
     androidxSqlite: '2.1.0',
     minSdk: 14,
     autoValue: '1.6.2',
     schemaCrawler: '14.16.04.01-java7',
-    sqldelight: '1.2.2',
+    sqldelight: '1.3.0-SNAPSHOT',
     moshi: '1.9.2',
-    stately: '0.9.5',
-    sqliter: '0.6.6',
+    stately: '1.0.0-a2',
```

Comment:
Will this cause issues as more alphas (assuming that's what "a" means) are released and folks wanna upgrade?

Reply:
Hmm. The "alpha" part of the library is really other unrelated modules, but yeah. I should have different versions for the stable modules.